### ISSUE TITLE
Fennec -> Fenix Data Migration UI

### ISSUE THREAD

**Issue by vesta0** [mozilla-mobile/fenix#934]
### User Story:
As a new non-FxA Fenix user, I want to the option of choosing what data to bring over from Fennec to Fenix and I want to know what to expect during the migration so I can have a smooth transition experience and not lose any data.

**Feature brief [here](https://docs.google.com/document/d/1zjWNE4oH8qkp6wV-nx_Nmi4_QfEBlsIyZ0bE71YfB_c/edit)**

### Reference Issues
Notifications: https://github.com/mozilla-mobile/fenix/issues/880

### Acceptance critera
-I can choose what data to bring over my Fennec data to Fenix
-I know what to expect before the migration

**Comment by lime124** [comment]
@vesta0 since this is a must for UX should we create another issue for that piece and place it in the must bucket? i'm worried that if my team is working off the must column in the q3 board we are going to miss this b/c it's not there.

**Comment by lime124** [comment]
removed ux label as it's on our radar as part of Q3 plans.

**Comment by liuche** [comment]
Keith Willowhawk is the PM for the Fennec Migration - @vesta0 will check with Keith about his UX resourcing for this plan.

**Comment by benryanau** [comment]
Assuming plans incorporate..: 
Opt-in UX decision reqd to initiate migration process.

**Comment by liuche** [comment]
@lime124 will get clear ordering of mobile PM priorities to get an idea of how to order UX priorities across different PMs.

**Comment by vesta0** [comment]
As discussed today, I will create a feature brief for this specific work. I also updated the User Story and Acceptance Criteria above for more clarify.

**Comment by lime124** [comment]
UX is waiting on the final decision for replace vs migrate from product (Reese).

**Comment by Moin-shadab** [comment]
Hlooo

**Comment by vesta0** [comment]
This ticket is no longer relevant since we will be doing the in-app replacement. All the related work are filed here #879 879
